clients/pkg/logentry/stages/replace.go

```diff
@@ -160,6 +161,7 @@ func (r *replaceStage) Process(labels model.LabelSet, extracted map[string]inter
 			}
 		}
 	}
+	level.Debug(r.logger).Log("msg", "extracted data debug in replace stage", "extracted data", fmt.Sprintf("%v", extracted))
```

Comment:
```
if Debug {
			level.Debug(r.logger).Log("msg", "failed to execute template on extracted value", "err", err)
}
```
We this Debug variable because the amount logs can be huge and level.Debug is expensive.